### PR TITLE
Remove root module.

### PR DESCRIPTION
**Pull request by omazin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +73,7 @@ module "storage_buckets_iam_bindings" {
 
 ### Variables
```
Comment (morgante) [inline README.md]: I think we could probably drop these variables? Or possibly just focus on an explanation of `Additive and Authoritative Modes` (combine with the section below).

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,13 @@ corresponding pull request appended.
 
 ## [Unreleased]
 
+## [5.0.0]
+
+This is a backward incompatible release. Refer to the [upgrade guide](docs/upgrading_to_iam_5.0.md) for more details.
+
+### Changed
+- The root module has been deprecated. [#73]
```
Comment (morgante) [inline CHANGELOG.md]:
```suggestion
- The root module has been removed. [#73]
```

File: docs/upgrading_to_iam_5.0.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+   kms_key_rings_bindings = {}
+   service_accounts_bindings = {}
+   folders_bindings = {}
+ }
+```
+
+## Upgrade Instructions
+
+The 5.0 release has deprecated the root module, so you should switch to a
+specific submodule, where settings defaults for *_bindings variables is dropped:
```
Comment (morgante) [inline docs/upgrading_to_iam_5.0.md]:
```suggestion
specific submodule, where setting defaults for *_bindings variables is not required:
```

File: README.md (hunk 1 above)
Comment (omazin) [reply README.md]: agreed

File: CHANGELOG.md (hunk 2 above)
Comment (omazin) [reply CHANGELOG.md]: fixed

File: docs/upgrading_to_iam_5.0.md (hunk 3 above)
Comment (omazin) [reply docs/upgrading_to_iam_5.0.md]: fixed

File: README.md [hunk 4]
```diff
@@ -82,96 +81,22 @@ module "storage_buckets_iam_bindings" {
...
-    This variable controls the module's behavior, by default is set to "additive", possible options are:
+### Additive and Authoritative Modes
 
-      - additive: add members to role, old members are not deleted from this role.
-      - authoritative: set the role's members, other roles' members are not deleted.
+The `mode` variable controls a submodule's behavior, by default it's set to "additive", possible options are:
 
-- Bindings
+  - additive: add members to role, old members are not deleted from this role.
+  - authoritative: set the role's members, other roles' members are not deleted.
```
Comment (morgante) [inline README.md]:
```suggestion
  - authoritative: set the role's members (including removing any not listed), unlisted roles are not affected
```

**Pull request merged**